contents/newsletter/small-teams.md

```diff
@@ -0,0 +1,118 @@
+
+1. Usage
+2. Quality
+3. Revenue
+
+A lot of the people on our small teams are product engineers – i.e. engineers who also talk to users, and own product decisions. And small teams aren't just for product – our marketing team is a small team, as are customer success, and people and operations. 
+
+This startup-made-of-startups structure minimizes the number of centralized processes and the need for lots of layers of management. It biases to [the maker’s schedule](https://www.paulgraham.com/makersschedule.html) – and makers get shit done.
+
+### They run themselves
+
+Each small team runs its own retrospective and sprint each week, with notes taken and shared in GitHub for the entire company to see.
+
+Small teams also make the final call on which features get into production, with no need for external quality assurance or control. And they can [merge whenever](https://posthog.com/handbook/engineering/development-process#merging).
+
+Each quarter, every small team at PostHog creates objectives and key results, or OKRs. Our executive team will give feedback on this to keep everyone on track. Beyond that, during each quarter each small team is tasked with maintaining a prioritized roadmap to help them achieve their objectives. Each small team is also tasked with speaking to customers. 
+
+And, as each one is like a startup in its own right, they’ve got to cover everything. That means:
+
+* Monitoring relevant metrics, including those covering usage, quality, and revenue
+* Triage and fix bugs related to the products or areas they’re responsible for
+* Assist the support hero in answering related questions
+* Collaborate with other Small Teams such as marketing
+* Become power users of their area of PostHog and use PostHog in their processes
+
+### They have one leader
+
+One of the purposes of the small team structure is to keep a startup’s structure flat even as it scales. That means [minimal layers of management](https://posthog.com/handbook/company/management) and lots of autonomy. 
+
+Each small team has a team lead who is responsible for its performance. It’s not a given that the lead is the most experienced person on the team, but rather the person best-suited to leading the product the team is working on. 
+
+At PostHog, we’ve got an engineering-led culture. And that means that our small teams are normally led by the person with overall responsibility for that product. And that person is very likely to be an engineer.
+
+Team leaders do not automatically equal managers. Team leads are responsible for making sure that teams perform well and for giving direct feedback. Managers have a different remit and are focused on the happiness of direct reports and setting the right context for people to do their jobs, onboarding new hires, and discussing performance issues with the executive team. A team lead’s remit is deliberately more product-focused.
+
+
+### They don’t clash
```

Comment:
I feel like this, or a version of it, might be better in the trade-offs section? In reality, the small teams do inevitably clash sometimes because some elements of the product are shared, it's who we deal with that counts.